caampr-backend/cloudformation/template.yml

```diff
@@ -70,22 +71,22 @@ Resources:
                   - logs:PutLogEvents
                 Resource: "*"
 
-# DynamoDB Resources
+  # DynamoDB Resources
   GearTable:
     Type: AWS::DynamoDB::Table
     DeletionPolicy: Retain
     Properties:
-      TableName: Gear-v2
+      TableName: Gear-v2b
       BillingMode: "PAY_PER_REQUEST"
       AttributeDefinitions:
-        - AttributeName: "id"
+        - AttributeName: "gearId"
           AttributeType: "S"
         - AttributeName: "type"
           AttributeType: "S"
         - AttributeName: "profileId"
           AttributeType: "S"
       KeySchema:
-        - AttributeName: "id"
+        - AttributeName: "gearId"
           KeyType: "HASH"
         - AttributeName: "type"
           KeyType: "RANGE"
```

Comment:
I thought we were gonna get rid of the GSI here since we aren't gonna query by index

Reply:
or i guess if everything is in lists prob not huh

Reply:
maybe for acct deletion we might need to remove all gear objects of a specific user regardless of list?

Reply:
ok, valid point about acct deletion and in case something becomes orphaned for any reason.  Let's keep it